src/pages/studyView/StudyViewPage.tsx

```diff
@@ -183,6 +199,26 @@ export default class StudyViewPage extends React.Component<
         this.props.routing.updateRoute({}, `study/${id}`);
     }
 
+    @observable showBookmarkModal = false;
+
+    @autobind
+    toggleBookmarkModal() {
```

Comment:
shouldn't this be annotated with `@action` annotation?

Reply:
yes